guides/05_chatbots/03-messages-format.md

```diff
@@ -0,0 +1,97 @@
+# Using the Messages data format
```

Comment:
The name of this file should be `03_messages-format.md` (the numerical prefix is followed by an underscore not a dash)

Reply:
🤦 Thank you!